webapp/webapp.spec.yaml

```diff
@@ -431,6 +431,26 @@ paths:
                 $ref: '#/components/schemas/VulnerabilitiesResponse'
       tags:
         - vulnerabilities
+
+  /v1/rpms:
+    post:
+      description: List of rpms by given srpm or rpm list and content set.
+      operationId: app.RpmsHandlerPost.post
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RpmsRequest'
+      responses:
+        '200':
+          description: List of rpms or content sets by given srpm or rpm list and content set.
```

Comment:
I would prefer more self-explanatory description that mentions clearly it is about `names`.
e.g. ```List of content sets with associated rpm names by given srpm names or list content sets by given rpm names.```